site/web/assets/css/librecores.css

```diff
@@ -519,3 +519,30 @@ h1 a:hover, h1 a:active {
 .bootstrap-select.btn-group .dropdown-menu {
   z-index: 1050; /* to over-ride navbar z-index*/
 }
+
+#delete-panel-button {
+  margin-left: 15px;
+}
+
+#delete-project-button {
```

Comment:
Do we really need this? It looks a bit out of place than the rest of the content

Reply:
For every element in a section there is a left-margin. If we see license, there is a left margin for license name field. It is the same case with other section elements. To keep it consistent with this format, i have put a margin. I shall decrease the margin and see if it looks better and consistent.

Reply:
I have reduced the margin. Does it look any better now ?

Reply:
Removed the margin.

Reply:
I feel, it looks better with a margin = 8px. Let me know what you think.